backend/src/tests/error_text.test.ts

```diff
@@ -0,0 +1,83 @@
+/* eslint-disable no-console */
+import supertest from 'supertest';
+import { app } from '../app';
+import { executeQuery } from '../db/queries';
+import { initializeUsers } from '../services/testService';
+
+const api = supertest(app);
+
+/**
+ * Tests the error messages
+ */
+
+beforeEach(async () => {
+    await initializeUsers();
+});
+
+describe('/user/register', () => {
+    test('The test user should be in the db', async () => {
+        const query = await executeQuery(
+            'SELECT * FROM users WHERE name = $1',
+            ['tester']
```

Comment:
Could this custom sql query be replaced by userExists from queries.ts?

Reply:
Yes it could, doesn't make any difference in sense of logic but I guess couple characters shorter code.